### PR TITLE
chore: Oct dependencies bump 🌲

### PR DESCRIPTION
**Pull request by stdavis**

### CODE REVIEW

**Review by steveoh — DISMISSED** [review]

**Review by steveoh — COMMENTED** [review]
should this go through dev to make sure the site loads alright?

**Comment by stdavis** [comment]
I was thinking that this repo would deploy to dev on any non-main branch. We need to come up with a more streamlined solution for dependabot PRs so that we don't double the number of PRs that we have to process going through dev and main. Maybe we can use deploy previews...

**Comment by stdavis** [comment]
@steveoh This is ready for another look.

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+    steps:
+      - name: 🚀 Deploy
+        uses: agrc/firebase-website-deploy-composite-action@v1
+        with:
+          identity-provider: ${{ secrets.IDENTITY_PROVIDER }}
+          service-account-email: ${{ secrets.SERVICE_ACCOUNT_EMAIL }}
+          project-id: ${{ secrets.PROJECT_ID }}
+          prebuild-command: npx grunt bump --setversion=${{ needs.release.outputs.released_version }}
+          build-command: npm run build:stage
```
Comment (steveoh) [inline .github/workflows/push.yml:65]: **issue**: missing repo token

File: .github/workflows/push.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+          identity-provider: ${{ secrets.IDENTITY_PROVIDER }}
+          service-account-email: ${{ secrets.SERVICE_ACCOUNT_EMAIL }}
+          project-id: ${{ secrets.PROJECT_ID }}
+          prebuild-command: npx grunt bump --setversion=${{ needs.release.outputs.released_version }}
+          build-command: npm run build:prod
+          service-now-instance: ${{ secrets.SN_INSTANCE }}
+          service-now-table: ${{ secrets.SN_TABLE }}
+          service-now-system-id: ${{ secrets.SN_SYS_ID }}
+          service-now-username: ${{ secrets.SN_USERNAME }}
+          service-now-password: ${{ secrets.SN_PASSWORD }}
```
Comment (steveoh) [inline .github/workflows/push.yml:89]: **issue**: missing repo token

**Review by steveoh — CHANGES_REQUESTED** [review]

File: .github/workflows/push.yml (hunk 1 above)
Comment (stdavis) [reply .github/workflows/push.yml:65]: I don't think that I need it here, do I? I guess it can't hurt...

**Review by steveoh — APPROVED** [review]

**Pull request merged**